faqs/galaxy/analysis_reporting_issues.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Reporting Issues and Feature Requests
+area: analysis
+box_type: tip
+layout: faq
+contributors: [jennaj, garimavs]
+---
+
+- For reporting **Usage Problems**, related to tools and functions, head to the [Galaxy Help](https://help.galaxyproject.org/) site.
+    - **Red error datasets**:
+        - To resolve it you might be asked to send a bug report.
+        - Click on the green bug icon within one of the red error datasets to start the submission process 
```

Comment:
Is it really green? the bug icon on failed datasets (at least on usegalaxy.eu) do not have any color.

Reply:
I actually wrote this with the help of the [Red error datasets](https://galaxyproject.org/issues/#red-error-datasets) and it has the green color mentioned in it. The icons used in GTN are all in black-white, but it won't be right to mention that icon as it will create chaos. Kindly correct me if I'm heading in the wrong direction.

Reply:
Some of these FAQs are old and may not be fully correct (sometimes interfaces have been updated). We shoudl reflect what we have today and check that what we write is correct today.

Reply:
Please insert icons whenever relevant, this will help guide the user. Here is a list of all available icons: https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#icons

so for the bug icon, you would use `{% icon galaxy-bug %}` next to where you say the words "bug icon"

If you ever find yourself wanting to use an icon you see in Galaxy, but not in the list I linked to above, let us know and we will add the icon!

Reply:
Thanks, @shiltemann, I have added the icon, but I haven't updated the commit yet so it is not visible. I will update so shortly.